src/Cmfd.cpp

```diff
@@ -45,7 +45,7 @@ Cmfd::Cmfd() {
   _group_indices_map = NULL;
 
   /* Initialize boundaries to be reflective */
-  _boundaries = new boundaryType[4];
+  _boundaries = new int[4];
```

Comment:
I must have missed where this change happened.  Why the change from enum to int?